addons/website_crm_partner_assign/static/src/js/crm_partner_assign.js

```diff
@@ -111,13 +111,13 @@ publicWidget.registry.crmPartnerAssign = publicWidget.Widget.extend({
      */
     _createOpportunity: function () {
         return this.orm.call("crm.lead", "create_opp_portal", [{
-            contact_name: $('.new_opp_form .contact_name').val(),
-            title: $('.new_opp_form .title').val(),
-            description: $('.new_opp_form .description').val(),
+            contact_name: document.querySelector(".new_opp_form .contact_name").value,
+            title: document.querySelector(".new_opp_form .title").value,
+            description: document.querySelector(".new_opp_form .description").value,
         }]).then(function (response) {
             if (response.errors) {
-                $('#new-opp-dialog .alert').remove();
-                $('#new-opp-dialog div:first').prepend('<div class="alert alert-danger">' + response.errors + '</div>');
+                document.querySelector("#new-opp-dialog .alert").remove();
```

Comment:
There might be no `.alert`: create opportunity then confirm without filling anything.